.github/workflows/lock.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-lock-inactive-days: '14'
+          pr-lock-inactive-days: '14'
```

Comment:
I'm -0 on locking PRs -- we don't actually have any issues with comments on old PRs afaik.

Reply:
If we're only locking issues, I think we need to `process-only: 'issues'` setting